### ISSUE TITLE
[Anon] Bug report: Dark Angels - Codex.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#386]
**File:** Dark Angels - Codex.catz

**BattleScribe version:** 1.14.07

**Platform:** iPhone / iPod / iPad

**Dropbox:** Yes

**Description:** I'm building a dark angels force and trying to add a land raider crusader.  

When I select it for 250 points, the notes say the loadout includes "Two flamestorm cannons" - shouldn't it be hurricane bolters?

Once selected when I proceed to the next page to specify the vehicle options, there's the option to select "land raider crusader" for another 280 points.  Shouldn't this be the 'deathwing vehicle' option?

Hope that helps

**Comment by Kangodo** [comment]
You are correct.
It seems that with the LRC Deathwing Vehicle is already selected.
And LR-Crusader and LR-Redeemer have their weapons swapped.
The LR-Redeemer also has the wrong point-cost for Deathwing Vehicle (it's free instead of 30).
Maybe @Thairne knows how this happened?

**Comment by Thairne** [comment]
I'm pretty sure I messed it up while redoing basically the entire catalogue and didn't catch that.
Seems to have been a copy&paste error and forgetting to fix them after correcting the deathwing transport entrys.
I'll get to it immediately.

Edit: Fixed it, together with the dedicated transport options and some missing wargear (frag launchers). Tested all 7 options in total (3 HS, all 3 variants as a dedicated transport for DWCS, DW Knights DW and DW Troops. Hope I caught everything this time).
Sorry for the inconvenience, being new to this and redesigning an entire catalogue unfortunately leads to this kind of errors :(
